test/e2e/install_test.go

```diff
@@ -27,62 +28,62 @@ var _ = Describe("Operator Install", func() {
 		operatorName string
 		operator     *operatorv1alpha1.Operator
 	)
+	When("An operator is installed from an operator catalog", func() {
+		BeforeEach(func() {
+			ctx = context.Background()
+			pkgName = "argocd-operator"
+			operatorName = fmt.Sprintf("operator-%s", rand.String(8))
+			operator = &operatorv1alpha1.Operator{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: operatorName,
+				},
+				Spec: operatorv1alpha1.OperatorSpec{
+					PackageName: pkgName,
+				},
+			}
+			operatorCatalog := &catalogd.CatalogSource{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-catalog",
+				},
+				Spec: catalogd.CatalogSourceSpec{
+					Image: "quay.io/operatorhubio/catalog:latest",
```

Comment:
This is a very large catalog to use in an e2e. Okay for now, but can you add a TODO comment and an issue in this repo to replace this with something we build locally/on-the-fly as part of the e2e test and make available during the e2e?

Reply:
Yup I'd discussed this yesterday and cc:@theishshah I'm adding the TODO and creating the issue so that we can keep track of this